package.json

```diff
@@ -6,35 +6,39 @@
   "scripts": {
     "install": "for d in packages/*/ ; do (cd \"$d\" && npm install); done",
     "flow": "flow",
-    "jest": "STORYBOOK_INLINE_APHRODITE=1 jest",
-    "test": "npm run flow && npm run jest",
-    "coverage": "STORYBOOK_INLINE_APHRODITE=1 jest --coverage",
-    "storybook": "start-storybook -p 9001 -c .storybook",
+    "gen-snapshot-tests": "node utils/gen-snapshot-tests.js",
```

Comment:
Could you add this to the pre-commit list, as well? (And git-add it, too, like with the docs.)

Reply:
There was already a `git-add-snapshot-tests` command that I had added, but I forgot to added to the list of `pre-commit` commands.  It's there now.